package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autotelic/fastify-hmac",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
NPM has v0.0.2 currently. I assume this should match so that the next patch update doesn't try to publish the same version number again.

Reply:
Good catch 👍